modules/i3s/src/lib/helpers/i3s-nodepages-tiles.ts

```diff
@@ -45,10 +45,10 @@ export default class I3SNodePagesTiles {
         this.options.i3s?.token
       );
       this.nodePages[pageIndex] = load(nodePageUrl, I3SNodePageLoader, this.options);
-      this.nodePages[pageIndex] = await this.nodePages[pageIndex];
+      this.nodePages[pageIndex] = this.nodePages[pageIndex];
     }
     if (this.nodePages[pageIndex] instanceof Promise) {
-      this.nodePages[pageIndex] = await this.nodePages[pageIndex];
+      this.nodePages[pageIndex] = this.nodePages[pageIndex];
```

Comment:
It is breaking change. `nodePages[index]` can be ongoing Promise or an object when it is loaded. We need to re-design this logic instead of just removing `await`

Reply:
Perhaps the easiest fix is just make sure the types is right. I removed the await because typescript told me to. So probably nodePages object just needs to be typed as `Promise<T> | T`

Reply:
Tests fail in this PR. I've done refactoring to fix this issue: https://github.com/visgl/loaders.gl/pull/1719